src/Sylius/Bundle/CoreBundle/CatalogPromotion/CommandDispatcher/BatchedApplyCatalogPromotionsOnVariantsCommandDispatcher.php

```diff
@@ -13,15 +13,23 @@
 
 namespace Sylius\Bundle\CoreBundle\CatalogPromotion\CommandDispatcher;
 
+use InvalidArgumentException;
```

Comment:
It's not set in automatic coding style for some reason, but we're following symfony CS, which states to not add `use` for global classes.

![image](https://github.com/Sylius/Sylius/assets/870747/b873d599-31b6-4f2c-8bbc-1fbba62ee85a)
https://symfony.com/doc/current/contributing/code/standards.html#structure